Source/WebSocket_Windows.cpp

```diff
@@ -0,0 +1,135 @@
+#include "WebSocket_Base.h"
+
+#define WIN32_LEAN_AND_MEAN
+
+#include <Windows.h>
+#include <winrt/Windows.Storage.Streams.h>
+#include <winrt/Windows.Networking.Sockets.h>
+#include <arcana/threading/task_conversions.h>
+
+namespace UrlLib
+{
+    using namespace winrt;
+
+    class WebSocket::WSImpl : public WSImplBase
+    {
+    public:
+        
+        void Send(std::string message)
+        {
+            if (m_readyState == ReadyState::Closed ||
+                m_readyState == ReadyState::Closing ||
+                m_readyState == ReadyState::Connecting)
+            {
+                error_callback();
+                throw std::runtime_error{"WebSocket is not Open"};
+            }
+
+            try
+            {
+                Windows::Storage::Streams::DataWriter dataWriter{m_webSocket.OutputStream()};
+                dataWriter.WriteString(winrt::to_hstring(message));
+
+                arcana::create_task<std::exception_ptr>(dataWriter.StoreAsync())
+                    .then(arcana::inline_scheduler, arcana::cancellation::none(), [this, dataWriter](int)
```

Comment:
You probably should have a cancellation source for these async tasks. Otherwise, there may be shutdown issues.